### PR TITLE
Set oc_share_external.name column length to 4000 to remove arbitrary …

### PR DESCRIPTION
**Pull request by come-nc**
…limitation

Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

**Review by szaimen — COMMENTED** [review]
increase the version number of files_sharing?

**Comment by szaimen** [comment]
@come-nc Was the 3rdparty bump intended?

**Comment by come-nc** [comment]
nope

**Review by blizzz — APPROVED** [review]

File: apps/files_sharing/lib/Migration/Version24000Date20220404142216.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		$table = $schema->getTable('share_external');
```
Comment (blizzz) [inline apps/files_sharing/lib/Migration/Version24000Date20220404142216.php:48]: wondering whether this table could be missing (defensive thinking), but seems quite quite unlikely.

Comment (come-nc) [reply apps/files_sharing/lib/Migration/Version24000Date20220404142216.php:48]: If it is missing something is off and we do want the migration to fail.

**Comment by come-nc** [comment]
Failures unrelated

**Comment by come-nc** [comment]
@PVince81 Should we backport this?

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
> @PVince81 Should we backport this?

this would likely make minor upgrade slower for not much benefit, so perhaps no
did we backport other ones like these ?

**Pull request merged**